apisix/cli/ops.lua

```diff
@@ -424,16 +429,44 @@ Please modify "admin_key" in conf/config.yaml .
                 port = 9090
             end
 
+            if ports_to_check[port] ~= nil then
+                util.die("control port conflicts with node_listen, prometheus, etc.\n")
```

Comment:
Better to use "control port xxx conflicts with node_listen, prometheus, etc."

Reply:
ok